dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.1.ee-7.0.feature

```diff
@@ -0,0 +1,20 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpTelemetry-1.1.ee-7.0
+singleton=true
+-features=\
+  com.ibm.websphere.appserver.javax.annotation-1.3, \
+  com.ibm.websphere.appserver.mpRestClient-2.0, \
+  com.ibm.websphere.appserver.mpConfig-2.0, \
+  com.ibm.websphere.appserver.cdi-2.0, \
+  io.openliberty.mpCompatible-4.0,\
```

Comment:
I don't think this is right, MP 4.0 is EE8. I think you wanted `mpCompatible-0.0` here.

Reply:
MP 1.4 is the last one which supports EE7...
```
"microProfile-1.4",
"servlet-3.1",
"cdi-1.2",
"jaxrs-2.0",
"jaxrsClient-2.0",
"jsonp-1.0",
"mpConfig-1.3",
"mpFaultTolerance-1.1",
"mpHealth-1.0",
"mpJwt-1.1",
"mpMetrics-1.1",
"mpOpenAPI-1.0",
"mpOpenTracing-1.1",
"mpRestClient-1.1"
```